README.md

```diff
@@ -505,6 +511,31 @@ td.inSignedMode() // true
 td.trackEvent('willbetracked') // will send td_ip and td_client_id; td_global_id will also be sent if set.
 ```
 
+### Treasure#fetchServerCookie(success, failure, forceFetch)
+
```

Comment:
Could you add a description of what this method should do -- like, why a customer would want to use it?